src/neo/Plugins/IPersistencePlugin.cs

```diff
@@ -35,8 +35,7 @@ public interface IPersistencePlugin
         /// </summary>
         /// <param name="system">The <see cref="NeoSystem"/> object that contains the blockchain.</param>
         /// <param name="block">The block being persisted.</param>
-        /// <param name="snapshot">The snapshot used for persistence.</param>
-        void OnCommit(NeoSystem system, Block block, DataCache snapshot) { }
+        void OnCommit(NeoSystem system, Block block) { }
```

Comment:
Won't think break existing persistence plugins that implement OnCommit?

Reply:
this will be part of 3.3?

Reply:
Yes.